### ISSUE TITLE
Couldn't communicate with the server

### ISSUE THREAD

**Issue by Stary4711** [xroix/MCBE-Win10-FOV-Changer#149]

**Comment by Yoda5577** [comment]
Hi! I'm having the same issue. I've restarted my computer, downloaded the most recent normal version, and even tried the beta but it hasn't worked. I launched it with administrator, and closed and opened both Minecraft and the FOV Changer multiple times. It only started occurring after I downloaded the most recent version, so I'm thinking Microsoft may have implemented more defenses against hacked clients, and that may have caused it to stop connecting now, I hope I've given enough information, and I also wanna say you did an amazing job with this.
